dataactbroker/handlers/fileHandler.py

```diff
@@ -1577,4 +1577,19 @@ def fabs_derivations(obj):
     federal_action_obligation = obj['federal_action_obligation'] or 0
     non_federal_funding_amount = obj['non_federal_funding_amount'] or 0
     obj['total_funding_amount'] = federal_action_obligation + non_federal_funding_amount
+
+    # deriving cfda_title from program_title in cfda_program table
+    sess = GlobalDB.db().session
```

Comment:
We should just put this at the top of the function because other derivations might need it and it's better practice to declare those kinds of variables at the top of the function.